src/main/resources/db/migration/V1_8__alter_audit_exception_table.sql

```diff
@@ -0,0 +1,10 @@
+truncate  dataload_schedular_audit;
+
+truncate dataload_exception_records;
+
```

Comment:
it is not in the prod right? ..can you update the existing script instead of creating new script and delete the resource group while merging the build? or else make

Reply:
Thats fine...it is not.Let it be incremental.

Reply:
Thats fine...it is not.Let it be incremental.